osu.Game/Screens/Select/Details/AdvancedStats.cs

```diff
@@ -122,32 +124,33 @@ private void updateStatistics()
                 case 3:
                     // Account for mania differences locally for now
                     // Eventually this should be handled in a more modular way, allowing rulesets to return arbitrary difficulty attributes
-                    firstValue.Title = "Key Count";
-                    firstValue.Value = (baseDifficulty?.CircleSize ?? 0, null);
+                    FirstValue.Title = "Key Count";
+                    FirstValue.Value = (baseDifficulty?.CircleSize ?? 0, null);
                     break;
 
                 default:
-                    firstValue.Title = "Circle Size";
-                    firstValue.Value = (baseDifficulty?.CircleSize ?? 0, adjustedDifficulty?.CircleSize);
+                    FirstValue.Title = "Circle Size";
+                    FirstValue.Value = (baseDifficulty?.CircleSize ?? 0, adjustedDifficulty?.CircleSize);
                     break;
             }
 
             starDifficulty.Value = ((float)(Beatmap?.StarDifficulty ?? 0), null);
 
-            hpDrain.Value = (baseDifficulty?.DrainRate ?? 0, adjustedDifficulty?.DrainRate);
-            accuracy.Value = (baseDifficulty?.OverallDifficulty ?? 0, adjustedDifficulty?.OverallDifficulty);
-            approachRate.Value = (baseDifficulty?.ApproachRate ?? 0, adjustedDifficulty?.ApproachRate);
+            HpDrain.Value = (baseDifficulty?.DrainRate ?? 0, adjustedDifficulty?.DrainRate);
+            Accuracy.Value = (baseDifficulty?.OverallDifficulty ?? 0, adjustedDifficulty?.OverallDifficulty);
+            ApproachRate.Value = (baseDifficulty?.ApproachRate ?? 0, adjustedDifficulty?.ApproachRate);
         }
 
-        private class StatisticRow : Container, IHasAccentColour
+        public class StatisticRow : Container, IHasAccentColour
         {
             private const float value_width = 25;
             private const float name_width = 70;
 
             private readonly float maxValue;
             private readonly bool forceDecimalPlaces;
             private readonly OsuSpriteText name, valueText;
-            private readonly Bar bar, modBar;
+            private readonly Bar bar;
+            public readonly Bar ModBar;
```

Comment:
As an aside, rather than exposing, you can use `ChildrenOfType<>()`. I'm neutral on whether this should be used in this case, up to you.

Reply:
I thought about it, since that's the way it was originally done, but it felt brittle due to relying on the implementation detail of order of addition. I do however agree that this exposition is a bit sketchy as well. Let me think this over a little bit.

Reply:
Fair, let's go with this for now.